doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -201,12 +201,26 @@ Automated metadata generation
 By enabling the ``SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA`` option in sysbuild, the metadata will be automatically parsed from the ``.elf`` and ``.config`` files.
 The parsed data is stored in the :file:`ble_mesh_metadata.json` file.
 The file is placed in the :file:`dfu_application.zip` archive in the build folder of the application.
-Additionally, the metadata string required by the ``mesh models dfu slot add`` command will be printed in the command line window when the application is built::
+Additionally, the metadata string and firmware ID required by the ``mesh models dfu slot add`` command is printed in the command-line window when the application is built::
 
   Bluetooth Mesh Composition metadata generated:
     Encoded metadata: 020000000000000094cf24017c26f3710100
+    Firmware ID: 59000200000000000000
     Full metadata written to: APPLICATION_FOLDER\build\zephyr\dfu_application.zip
 
+The Firmware ID can be generated from one of multiple sources:
```

Comment:
Something seems strange to read here.

```suggestion
The Firmware ID can be generated from one of the multiple sources:
```
or

```suggestion
The Firmware ID can be generated using one of the options mentioned below:
```

Reply:
"The Firmware ID can be generated from one of _the_ multiple sources" is objectively worse grammar, isn't it? @peknis ?

Reply:
Yes, the article was not meant to be there. Omkar's suggestion sounds better.